setup.py

```diff
@@ -29,4 +29,5 @@ def run(self):
         ('lib/systemd/user-generators', ['systemd-procserv-generator-user']),
     ],
     cmdclass = { 'install_data': custom_install_data },
+    #install_requires=['argcomplete', 'tabulate', 'termcolor'],
```

Comment:
If these are to be unconditional dependencies, then `install_requires` needs to be uncommentated.

Reply:
The problem here is that the project uses ``distutils`` to install the python scripts, which does not support the ``install_requires`` directive. Maybe it's possible to use ``pip``, but I'm not sure how to integrate it with the makefiles.

For the moment I will follow your suggestion to make this packages optional using the ``try: ... import`` mechanism and defining some fallback behaviors.

Reply:
Ya...  I was a late and reluctant convert to using setuptools.  I had hoped that distutils would be cleaned up to include the useful parts.  However, the opposite has happened.  With so much external tooling with invasive hooks into distutils, it doesn't seem like it can reasonable be changed.  At least setuptools availability is ubiquitous now.

Although I don't think this module will ever be usable in eg. a virtualenv, switching to setuptools is probably reasonable.  Doing this may necessitate changes to the `custom_install_data` though.